README.md

```diff
@@ -73,15 +69,11 @@ jobs:
 
       - uses: im-open/octostache-action@v1.0.0
         with:
-          variablesFile: ./substitution-variables.json
-          templateFile: ./src/DemoApp19/DemoApp19.csproj
-          outputFile: ./src/DemoApp19/DemoApp19.csproj
-
-      - uses: im-open/octostache-action@v1.0.0
-        with:
-          variablesFile: ./substitution-variables.json
-          templateFile: ./src/DemoApp19/Bff/FrontEnd/scripts/build-variables.js
-          outputFile: ./src/DemoApp19/Bff/FrontEnd/scripts/build-variables.js
+          variables-file: ./substitution-variables.json
+          files-with-substitutions: ./src/DemoApp19/DemoApp19.csproj,./src/DemoApp19/Bff/FrontEnd/scripts/build-variables.js
+          output-files: ./src/DemoApp19/DemoApp19.csproj,./src/DemoApp19/Bff/FrontEnd/scripts/build-variables.js
+        env:
+          LaunchDarklyKey: ${{ secrets.LAUNCH_DARKLY_API_KEY }}
```

Comment:
This might be a scenario you want to cover in the readme, what takes precedence if you have it in the file and in an environment variable.